### DISCUSSION TITLE
v4.0.0

### DISCUSSION THREAD

**Discussion by JoseRamosMendez** [OpenTOPAS/OpenTOPAS#9] [Announcements]
Compatible with Geant4.11.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenTOPAS/OpenTOPAS/releases/tag/'>v4.0.0</a>.</em>

**Comment by claus-e-andersen** [comment]
I can confirm that the new code calls: 
Geant4 version Name: geant4-11-01-patch-03 [MT]   (10-November-2023)
However, it appears that TOPAS is still at version 3.9.0 (not 4.0.0):
```shell
set (TOPAS_VERSION_MAJOR 3)
set (TOPAS_VERSION_MINOR 9)
set (TOPAS_VERSION_PATCH 0)
``` 
See file : https://github.com/OpenTOPAS/OpenTOPAS/tree/master)
 /CMakeLists.txt


- Claus

**Reply by claus-e-andersen** [reply]
I see that the version number has now been fixed in the branch called versionfix. Likevwise, the errors in the linux installation instructions have also been fixed for this branch. However, new users that will download from the menu of your web page will be guided by the instructions in the master branch, and can therefore not easily benefit from these improvements. Would it be possible to link new users directly to the versionfix branch.     

The development branch also has version number 3.9.0.

- Claus

**Reply by JoseRamosMendez** [reply]
To protect the master branch, we adopted a pull-request -> review -> merge workflow (even for admin). Now the commit is under review, and soon will be merged to the master (but cannot rush the reviewer on Sunday :) )

**Reply by claus-e-andersen** [reply]
Ok, I get it. Thanks.

**Reply by JoseRamosMendez** [reply]
Thanks for the help!
